lib/new_relic/agent/llm/llm_event.rb

```diff
@@ -19,6 +19,12 @@ class LlmEvent
         INGEST_SOURCE = 'Ruby'
         LLM = :llm
         CUSTOM_ATTRIBUTE_CONVERSATION_ID = 'llm.conversation_id'
+        ERROR_ATTRIBUTE_STATUS_CODE = 'http.statusCode'
+        ERROR_ATTRIBUTE_CODE = 'error.code'
+        ERROR_ATTRIBUTE_PARAM = 'error.param'
+        ERROR_STRING = 'error'
+        CODE_STRING = 'code'
+        PARAM_STRING = 'param'
```

Comment:
Since these constants are used by both Embedding and ChatCompletionSummary, I thought I'd put them here. In the future, if we work on https://github.com/newrelic/newrelic-ruby-agent/issues/2436, we might want to move these constants into that shared folder.